src/main/java/com/marklogic/mapreduce/test/CustomQuery.java

```diff
@@ -17,7 +17,7 @@
 
 import java.io.IOException;
 
-import javax.xml.soap.Node;
+import jakarta.xml.soap.Node;
```

Comment:
Seems like this class should be moved to src/test/java if it's intended to be a test?